x-pack/plugins/ml/public/application/components/ml_inference/components/test_pipeline.tsx

```diff
@@ -202,12 +329,24 @@ export const TestPipeline: FC<Props> = memo(({ state, sourceIndex }) => {
               <EuiFlexItem grow={false}>
                 <EuiButtonEmpty
                   size="xs"
-                  onClick={getSampleDocs}
+                  onClick={() => setSampleDocsString(lastFetchedSampleDocsString)}
                   disabled={sampleDocsString === ''}
                 >
                   {i18n.translate(
                     'xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.test.resetSampleDocsButton',
-                    { defaultMessage: 'Reset sample docs' }
+                    { defaultMessage: 'Reset' }
```

Comment:
Is it possible to grey out the Reset button until a change is made to the document? With the addition of the Reload button, it wasn't immediately clear to me what the Reset action was doing.

Reply:
Updated in 2c51898d8c7022455d87e5815e92eb7880e092f9